### ISSUE TITLE
Domain-specific Batch-Normalization

### ISSUE THREAD

**Issue by szubing** [VisionLearningGroup/DANCE#3]
In the page of 3, in section 3, the authors have said that "In addition, we utilize domain-specific" batch normalization to ..."; However, I do not found that the code is implemented using domain-specific batch normalization. So I am a little curious about that? Can you explain why?

**Comment by ksaito-ut** [comment]
We used a simplified version of the domain-specific batch norm. The source and target mini-batches are fed into the network separately.
